src/base-commands/twilio-client-command.js

```diff
@@ -111,7 +111,8 @@ class TwilioClientCommand extends BaseCommand {
   buildClient(ClientClass) {
     return new ClientClass(this.currentProfile.apiKey, this.currentProfile.apiSecret, {
       accountSid: this.flags[CliFlags.ACCOUNT_SID] || this.currentProfile.accountSid,
-      region: this.currentProfile.region,
+      edge: process.env.TWILIO_EDGE || this.userConfig.edge,
+      region: process.env.TWILIO_REGION || this.currentProfile.region,
```

Comment:
I don't think we should pull the region env var here. Since region is tied to auth, the profile object should contain the correct region already.

Reply:
Ah, forgot it was pulled when it pulls the other profile info from env vars.